slycot/synthesis.py

```diff
@@ -2448,3 +2448,260 @@ def sg03bd(n,m,A,E,Q,Z,B,dico,fact='N',trans='N',ldwork=None):
+    A : (n, n) array_like
+    B : (n, m) array_like
+    C : (np, n) array_like
+    D : (np, m) array_like
+    tol : float, optional
+        Tolerance used for controlling the accuracy of the applied
+        transformations for computing the normalized form in
+        SLICOT Library routine SB10PD. Transformation matrices
+        whose reciprocal condition numbers are less than tol are
+        not allowed. If tol <= 0, then a default value equal to
+        sqrt(eps) is used, where eps is the relative machine
+        precision.
+    ldwork : int, optional
+        The dimension of the cache array.
+        ldwork >= n*m + np*(n+m) + m2*m2 + np2*np2 +
+                   max(1,lw1,lw2,lw3,lw4,lw5,lw6), where
+        lw1 = (n+np1+1)*(n+m2) + max(3*(n+m2)+n+np1,5*(n+m2)),
+        lw2 = (n+np2)*(n+m1+1) + max(3*(n+np2)+n+m1,5*(n+np2)),
+        lw3 = m2 + np1*np1 + max(np1*max(n,m1),3*m2+np1,5*m2),
+        lw4 = np2 + m1*m1 + max(max(n,np1)*m1,3*np2+m1,5*np2),
+        lw5 = 2*n*n + n*(m+np) +
+              max(1,m*m + max(2*m1,3*n*n+max(n*m,10*n*n+12*n+5)),
+                  np*np + max(2*np1,3*n*n +
+                              max(n*np,10*n*n+12*n+5))),
+        lw6 = 2*n*n + n*(m+np) +
+              max(1, m2*np2 + np2*np2 + m2*m2 +
+                  max(d1*d1 + max(2*d1, (d1+d2)*np2),
+                      d2*d2 + max(2*d2, d2*m2), 3*n,
+                      n*(2*np2 + m2) +
+                      max(2*n*m2, m2*np2 +
+                          max(m2*m2+3*m2, np2*(2*np2+
+                              m2+max(np2,n)))))),
+        with d1 = np1 - m2, d2 = m1 - np2,
+            np1 = np - np2, m1 = m - m2.
+        For good performance, ldwork must generally be larger.
+        Denoting q = max(m1,m2,np1,np2), an upper bound is
+        2*q*(3*q+2*n)+max(1,(n+q)*(n+q+6),q*(q+max(n,q,5)+1),
+        2*n*(n+2*q)+max(1,4*q*q+
+                        max(2*q,3*n*n+max(2*n*q,10*n*n+12*n+5)),
+                            q*(3*n+3*q+max(2*n,4*q+max(n,q))))).
+        if the default (None) value is used, the size for good performance
+        is automatically used, when ldwork is set to zero, the minimum
+        cache size will be used.
+
+    Returns
+    -------
+    Ak : (n, n) ndarray
+        The controller state matrix Ak.
+    Bk : (n, nmeas) ndarray
+        The controller input matrix Bk.
+    Ck : (ncon, n) ndarray
+        The controller output matrix Ck.
+    Dk : (ncon, nmeas) mdarrau
+        The controller input/output matrix Dk.
+    rcond : (4, ) ndarray
+        rcond[1] contains the reciprocal condition number of the
+                 control transformation matrix
+        rcond[2] contains the reciprocal condition number of the
+                 measurement transformation matrix
+        rcond[3] contains an estimate of the reciprocal condition
+                 number of the X-Riccati equation
+        rcond[4] contains an estimate of the reciprocal condition
+                 number of the Y-Riccati equation
```

Comment:
Python vs Fortran indexing. The second line indented also looks a bit ugly when rendered:

![Screenshot_20200511_170337](https://user-images.githubusercontent.com/4623504/81577250-7f235700-93a9-11ea-8b06-bf85e67bbef8.png)


```suggestion
    rcond : (4, ) ndarray
        rcond[0]
                 contains the reciprocal condition number of the
                 control transformation matrix
        rcond[1]
                 contains the reciprocal condition number of the
                 measurement transformation matrix
        rcond[2]
                 contains an estimate of the reciprocal condition
                 number of the X-Riccati equation
        rcond[3]
                 contains an estimate of the reciprocal condition
                 number of the Y-Riccati equation
```